varcode/frameshift_coding_effect.py

```diff
@@ -118,8 +118,84 @@ def _frameshift(
         transcript=transcript,
         aa_mutation_start_offset=mutation_start_position,
         aa_ref=aa_ref,
-        shifted_sequence=mutant_protein_suffix)
+        shifted_sequence=str(mutant_protein_suffix))
 
+def cdna_codon_sequence_after_insertion_frameshift(
```

Comment:
Yeah, I struggled with how to name these functions. It's the cDNA sequence starting from the first mutant codon (even if that includes 1 or 2 reference nucleotides before the insertion). Any ideas for a better name?